src/librustc_driver/Cargo.toml

```diff
@@ -12,8 +12,8 @@ crate-type = ["dylib"]
 [dependencies]
 lazy_static = "1.0"
 libc = "0.2"
-log = "0.4"
-env_logger = { version = "0.7", default-features = false }
+log = { package = "tracing", version = "0.1" }
```

Comment:
The replacements seem like they could be pretty confusing, so we should try to move off of them pretty quickly, but I agree that for now this is probably the right way to do things, presuming it doesn't cause problems with tracing's own macros expanding to `::tracing::foo` and that not resolving.

Reply:
I agree, I'll open an E-easy issue with checkboxes for each of the renames

Reply:
Tracing's macros should all use `$crate::`, so this should be fine.